developer/tryout/samples/sample-ratelimit.yaml

```diff
@@ -3,11 +3,11 @@ kind: RateLimitPolicy
 metadata:
   name: sand-http-bin-ratelimit
 spec:
-  policies:
-    - type: Api
-      api:
-        requestPerUnit: 5
-        unit: Minute
+  override:
+    type: Api
+    api:
+      count: 5
```

Comment:
Just saying my opinion, I think `requestPerUnit` and `unit` are clearer than `count` and `spanUnit` terms

Reply:
Yes, I too agree.